src/builder-module.c

```diff
@@ -1741,6 +1761,10 @@ builder_module_build_helper (BuilderModule  *self,
               configure_cmd = "meson";
               configure_final_arg = g_strdup ("..");
             }
+          else if (cargo)
+            {
+              // nothing to do - Cargo does not have a configure step
```

Comment:
so how do you select features?
or building a specific binary.

Reply:
Good point. Just to be more specific about this, do you mean switches like:

- Profile: `cargo build --release / --debug`
- Target: `cargo build --target <architecture>`
- Cargo features: `cargo build --features <feature>,<feature>,...`